modules/bim/config/locales/js-en.yml

```
@@ -14,6 +14,7 @@ en:
       management: 'BCF management'
       refresh: 'Refresh'
       refresh_work_package: 'Refresh work package'
+      snapshot_alt: 'BCF snapshot'
```

Please do that as one string so that it does not have to be concatinated all the time.
```suggestion
      snapshot_alt: 'BCF snapshot for work package %{title}'
```